### ISSUE TITLE
Attacking Google Cloud Vision API

### ISSUE THREAD

**Issue by Eric-mingjie** [cg563/simple-blackbox-attack#5]
I was using your code (simba_single.py) in the github repo to attack the google cloud vision api. However, I found that the probability score of the top class stays exactly the same for initial 20 iterations. (I only tried 20 iterations as I want to save my quota). It seems that the attack becomes stuck right at the beginning. What is the problem here?

**Comment by cg563** [comment]
We did not encounter this problem when running our attack. Could you please confirm that the code runs fine on a local model?

One possible explanation is that the step size you used is too small, and the output image file quantized it to zero. Maybe you can try the GCV attack again with a larger step size?

**Comment by Eric-mingjie** [comment]
Thanks for your timely response.
1. I have tried running the attack against an ImageNet model first. Everything works well.

2. I did check the output image file, it is different from the original clean image with step size 0.2.  

3. The step size I tried was 0.2, which I think is the value you reported in your paper. I also tried step size 0.4. The score of the top class still remains constant.
